packages/devextreme/js/__internal/ui/chat/chat_message_list.ts

```diff
@@ -60,7 +61,10 @@ class MessageList extends Widget<MessageListOptions> {
   }
 
   _createMessageGroupComponent(items: Message[], userId: string | number | undefined): void {
-    const $messageGroup = $('<div>').appendTo(this._$content);
+    const scrollableContent = $(this.$element()).find(`.${SCROLLABLE_CONTENT_CLASS}`).get(0);
+    const messageGroupContainer = scrollableContent || this._$content;
```

Comment:
const $messageGroupContainer = this._scrollable
    ? this._scrollable.content()
    : this._$content;

Reply:
Don't forget to remove scrollable on call _clean() method.
```
    this._scrollable = undefined;
    delete this._scrollable;
```
Let's make `private _scrollable?: Scrollable<unknown>;` optional, because exist cases when it's not exist

Reply:
[d2c631211da8af6d62c8b2788168dde7ea574415]